common/txmgr/tracker.go

```diff
@@ -115,6 +114,8 @@ func (tr *Tracker[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) startIntern
 		tr.lggr.Infow("no abandoned txes found, skipping runLoop")
 		return nil
 	}
+
+	tr.lggr.Infow(fmt.Sprintf("%d abandoned txes found, starting runLoop", len(tr.txCache)))
```

Comment:
nit: This can probably be simplified down to something like

`tr.lggr.Infof("%d abandoned txes found, starting runLoop", len(tr.txCache))`